hugolib/pageGroup_test.go

```diff
@@ -149,7 +149,7 @@ func (page *Page) DummyPageMethodReturnThreeValueForTest() (string, string, stri
 }
 
 func (page *Page) DummyPageMethodReturnErrorOnlyForTest() error {
-	return errors.New("something error occured")
+	return errors.New("something error occurred")
```

Comment:
If you want to fix this and the similar below, you should fix both language errors in the same sentence.